### PR TITLE
Validate content type removal

### PR DESCRIPTION
**Pull request by bratseth**
@geirst please review

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidator.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline config-model/src/main/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidator.java]: Please update year

File: config-model/src/main/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidator.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import com.yahoo.documentmodel.NewDocumentType;
+import com.yahoo.vespa.model.VespaModel;
+import com.yahoo.vespa.model.content.cluster.ContentCluster;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Checks that this does not remove a data type in a cluster, at that causes deletion
```
Comment (geirst) [inline config-model/src/main/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidator.java]: at -> as ?

File: config-model/src/test/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidatorTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline config-model/src/test/java/com/yahoo/vespa/model/application/validation/change/ContentTypeRemovalValidatorTest.java]: Please update year

**Review by geirst — COMMENTED** [review]
Overall looks good, please consider comments

**Comment by bratseth** [comment]
Both done - thanks.

**Comment by geirst** [comment]
LGTM

**Pull request merged**